lncfg/db.go

```diff
@@ -55,6 +55,11 @@ func (db *DB) GetBackend(ctx context.Context, dbPath string,
 	networkName string) (kvdb.Backend, error) {
 
 	if db.Backend == etcdBackend {
+		if db.Etcd.Embedded {
+			db, _, err := kvdb.GetEtcdTestBackend(dbPath, dbName)
```

Comment:
I think we should pass in the main `etcd` config here too so it can start with the proper settings for transaction size, etc.